### DISCUSSION TITLE
How to resize QAvatar dynamically

### DISCUSSION THREAD

**Discussion by iamshulz** [quasarframework/quasar#9623] [General - Components / Directives / etc]
I have this q-avatar that I want to resize dynamically. I'm able to get the width of the window and do my own calculations to be set on `q-avatar` based on the width of the browser
using `:size` does not work and it gives me error I have also used the q-image but the problem with q-image is that the image will not adjust according to avartar size so the image will overlap which is weird.
```html
 <div class="q-pa-xs row">
      <q-card-section class="col-sm-12 col-md-6 text-center">
          <q-avatar size="250px"> <!-- ! here ! -->
                        <!-- <q-img src="https://cdn.quasar.dev/img/mountains.jpg" sizes="(max-width: 400px) 400w"></q-img> -->
                        <img src="https://cdn.quasar.dev/img/mountains.jpg">
                    </q-avatar>
                    <q-card-section>
                        <div class="text-h6">Our Changing Planet windowSize: {{ windowSize }} </div>
                        <div class="text-subtitle2">by John Doe</div>
           </q-card-section>
     </q-card-section>
</div>
```

**Comment by hawkeye64** [comment]
@jamols09 Put this into a CodePen and someone will be happy to help you.

**Comment by FelixDKBinder** [comment]
@jamols09  Maybe the [responsive](https://quasar.dev/vue-components/responsive#example--on-qcardsection) component can help you with that
